src/taco-dependency-installer/utils/win32/elevatedInstallerLauncher.ps1

```diff
@@ -4,7 +4,7 @@ $pinfo.FileName = "node.exe"
 $pinfo.UseShellExecute = $true
 $pinfo.WindowStyle = "Hidden"
 $pinfo.Verb = "RunAs"
-$pinfo.Arguments = "$($args[0]) $($args[1]) $($args[2]) $($args[3])"
+$pinfo.Arguments = "$args
```

Comment:
Looks like this is broken; I assume it should be passing through another argument or so rather than removing a bunch?